raft/server.cc

```diff
@@ -431,7 +431,7 @@ future<> server_impl::wait_for_next_tick(seastar::abort_source* as) {
     try {
         co_await (as ? _tick_promise->get_shared_future(*as) : _tick_promise->get_shared_future());
     } catch (abort_requested_exception&) {
-        throw request_aborted();
+        throw request_aborted("Cancellation requested via abort_source::request_abort() while waiting for next tick");
```

Comment:
Is it necessary to specify the `abort_source::request_abort()` in the messages? (maybe it can be omitted)

Reply:
removed